internal/static/integrations/ssh_exporter/ssh_client.go

```diff
@@ -0,0 +1,194 @@
+package ssh_exporter
+
+import (
+    "bytes"
+    "fmt"
+    "os"
+    "path/filepath"
+    "time"
+    "os/exec"
+    "strings"
+
+    "github.com/go-kit/log"
+    "github.com/go-kit/log/level"
+    "golang.org/x/crypto/ssh"
+    "golang.org/x/crypto/ssh/knownhosts"
+    "os/user"
+)
+
+
+type SSHClient struct {
+    config  *ssh.ClientConfig
+    host    string
+    port    int
+    logger  log.Logger
+    timeout time.Duration
+}
+var sshKeyscanCommand = func(targetAddress string) ([]byte, error) {
+    cmd := exec.Command("ssh-keyscan", "-H", targetAddress)
```

Comment:
Can target address be hijacked for any injection? We should ensure it is an valid URI.